pillar/heroku/discussions.sls

```diff
@@ -129,6 +135,9 @@
       'TIKA_SERVER_ENDPOINT': 'https://tika-production.odl.mit.edu',
       'vault_env_path': 'production-apps',
       'env_stage': 'production',
+      'FEATURE_KEYCLOAK_ENABLED': False,
+      'KEYCLOAK_BASE_URL': '',
+      'KEYCLOAK_REALM_NAME': '',
```

Comment:
These values need to be added to the body of the YAML document in order for them to actually render to Heroku.